app/src/main/java/nextstep/jwp/http/response/HttpResponse.java

```diff
@@ -0,0 +1,85 @@
+package nextstep.jwp.http.response;
+
+import java.io.IOException;
+import nextstep.jwp.http.common.HttpHeaders;
+import nextstep.jwp.http.common.ResourceFile;
+
+public class HttpResponse {
+
+    private static final String NEWLINE = "\r\n";
+    private static final String HTTP_VERSION = "HTTP/1.1 ";
+    private static final String CONTENT_TYPE = "Content-Type";
+    private static final String CONTENT_LENGTH = "Content-Length";
+
+
+    private HttpStatus httpStatus;
+    private HttpHeaders httpHeaders;
+    private String body = "";
+
+    public HttpResponse() {
+        this.httpHeaders = HttpHeaders.of();
+    }
+
+    public HttpResponse(HttpStatus httpStatus, HttpHeaders httpHeaders, String body) {
+        this.httpStatus = httpStatus;
+        this.httpHeaders = httpHeaders;
+        this.body = body;
+    }
+
+    public void setHttpStatus(HttpStatus httpStatus) {
+        this.httpStatus = httpStatus;
+    }
+
+    public void addHeader(String key, String value) {
+        httpHeaders.addHeader(key, value);
+    }
+
+    public void setBody(String body) {
+        this.body = body;
+    }
+
+    public HttpStatus getHttpStatus() {
+        return httpStatus;
+    }
+
+    public String getBody() {
+        return body;
+    }
+
+    public byte[] getResponseByByte() {
+        return String.join(NEWLINE,
+                getHttpLine(),
+                httpHeaders.convertToLines(),
+                "",
+                body
+        ).getBytes();
+    }
+
+    private String getHttpLine() {
+        return HTTP_VERSION + httpStatus.value() + " " + httpStatus.responsePhrase() + " ";
```

Comment:
```suggestion
        return String.format("%s %s %s ", HTTP_VERSION, httpStatus.value(), httpStatus.responsePharse());
```
이런 방법을 써도 좋을 것 같네요! 개인 취향이니 넘어가도 좋습니다ㅎㅎ